Cargo.toml

```diff
@@ -73,7 +73,7 @@ cdn-marshal = { git = "https://github.com/EspressoSystems/Push-CDN", features =
   "global-permits",
 ], tag = "0.4.2", package = "cdn-marshal" }
 
-jf-plonk = { git = "https://github.com/EspressoSystems/jellyfish", tag = "jf-plonk-v0.5.0", features = [
+jf-plonk = { git = "https://github.com/EspressoSystems/jellyfish", branch = "cp-m04", features = [
```

Comment:
I'll update this once the cooresponding PR on jellyfish is merged. 

at the end of the re-review, we will merge `commonprefix-patch` branch on jellyfish into `main` and release a new tag. and update here.